src/routes/subscribe/add.ts

```diff
@@ -10,7 +11,7 @@ const addSubscriber: RequestHandler = async (req, res) => {
 
     // Check if email is provided
     if (!eAddress) {
-      return res.status(400).json({
+      return res.status(400).j({
         status: 'error',
         message: 'No email provided',
```

Comment:
move this check before `eAddress = email.toLowerCase()` because of `undefined.toLowerCase()` then error throws.